cmake/modules/MacOSXBundleInfo.plist.in

```diff
@@ -118,5 +118,13 @@
 	<true/>
 	<key>NSHumanReadableCopyright</key>
 	<string>${MACOSX_BUNDLE_COPYRIGHT}</string>
+	<key>NSAppTransportSecurity</key>  
+	<dict>
+		<!-- Disable App Transport Security. Resolves https://github.com/SoftFever/OrcaSlicer/issues/791 -->
+		<key>NSAllowsArbitraryLoads</key>
+		<true/>  
+		<key>NSAllowsArbitraryLoadsInWebContent</key>
+		<true/>
+	</dict>
```

Comment:
This is the change that ultimately resolves the issue.

Reply:
@ammmze, while I was Googling the NSAppTransportSecurity thing, I came across an interesting key called [NSAllowsLocalNetworking](https://developer.apple.com/documentation/bundleresources/information_property_list/nsapptransportsecurity/nsallowslocalnetworking). 

> set both of the [NSAllowsArbitraryLoads](https://developer.apple.com/documentation/bundleresources/information_property_list/nsapptransportsecurity/nsallowsarbitraryloads) and NSAllowsLocalNetworking keys to YES

If I read it correctly, it seems like a better approach as it allows disabling ATS for the local network while still enforcing it for public domains. 
Would you be willing to give it a try?

Reply:
👍🏻 Let me give it a go

Reply:
I tried setting that, but unfortunately it did not work for me.

```xml
	<key>NSAppTransportSecurity</key>
	<dict>
		<key>NSAllowsLocalNetworking</key>
		<true/>
	</dict>
```

Reply:
You need to set both of the [NSAllowsArbitraryLoads](https://developer.apple.com/documentation/bundleresources/information_property_list/nsapptransportsecurity/nsallowsarbitraryloads) and NSAllowsLocalNetworking keys to YES

Reply:
Gotcha ... Not sure if its just because I am using a real TLD for my network domain but for me only `NSAllowsArbitraryLoads` allows it work. And setting that is what effectively disables ATS.

Reply:
Oh I see, the combination of them disables it for local networking only. Still doing some experimentation...it might only work for `*.local` domains because it didn't work for my TLD.

Reply:
Yea, it looks like that works for `*.local` domains, but not for other TLDs like `.lan` and not for real TLD's like I use. After finding that out, I went back and tested without `NSAllowsArbitraryLoads` and `printer.local` worked there. So it looks like just adding `NSAllowsLocalNetworking` will allow it to work for users that use `*.local`. 

But for other domains we need to either:
1. Completely disable ATS (which is not recommended)
2. Add domain exception(s) (either user does this manually, or we try to edit the plist in the app) and restart the app.

Reply:
I also see it that way.

Reply:
> Yea, it looks like that works for `*.local` domains, but not for other TLDs like `.lan` and not for real TLD's like I use. After finding that out, I went back and tested without `NSAllowsArbitraryLoads` and `printer.local` worked there. So it looks like just adding `NSAllowsLocalNetworking` will allow it to work for users that use `*.local`.
> 
> But for other domains we need to either:
> 
> 1. Completely disable ATS (which is not recommended)
> 2. Add domain exception(s) (either user does this manually, or we try to edit the plist in the app) and restart the app.

I thought .local covers all the local domains in the LAN, no?
Wondering what other domain are you referring to?

Reply:
I believe a lot of routers default to using `.local` and the network domain. And as such devices that publish their hostname as "printer", for example they can be found on the local network as `printer.local`. In addition mDNS also uses the `.local` domain. However many people will change their network domain to something like `.lan` or some may use a real domain they have purchased. I personally use a domain I have purchased as I can then generate valid SSL certs from let's encrypt (ironic since I have not done that here yet but that would solve the issue for me). 

So anyways, we can make it work for `.local` domains with this flag while still keeping ATS for everything else if that's what we want to do.